hr_expense_cancel/README.rst

```diff
@@ -0,0 +1,108 @@
+    :target: https://translation.odoo-community.org/projects/hr-13-0/hr-13-0-hr_expense_cancel
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/116/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows you to cancel and correct expenses. It adds a cancel button
+on the expense sheet that undo reconciliations and delete payments and journal
+entries.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+It's assumed that for each expense sheet, the *payments* that are reconciled
+with their *journal entry* were created and reconciled by the *Expense Tracker*
+module automatically. Therefore, when this module is installed, these payments
+will automatically be associated with said expense sheet so that they can be
+deleted in case the expense sheet is canceled.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to *Expenses -> My Expenses -> Expenses to Submit* and create a new
+   *Expense* with *Employee (to reimburse)* checked on the field *Payment By*
+#. Click on *Submit to Manager* button
+#. Click on *Approve* button
+#. Click on *Post Journal Entries* button
+#. Click on *Register Payment* button, fill in the data of the wizard and
+   click on *Validate* button
+#. After that, the *Expense report* will have an associated journal entry
+   reconciled with a payment
+#. Click on *Cancel* button
+#. The *Expense report* will be set to *Submitted* state; the journal entry and
+   the payment will be deleted
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/hr/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/hr/issues/new?body=module:%20hr_expense_cancel%0Aversion:%2013.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Tecnativa
+
+Contributors
+~~~~~~~~~~~~
+
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Pedro M. Baeza
+  * Ernesto Tejeda
+
+* Danh Vo  <https://github.com/danhvophuong>
+* Saran Lim. <saranl@ecosoft.co.th>
+
+Maintainers
+~~~~~~~~~~~
+
+This module is maintained by the OCA.
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/hr <https://github.com/OCA/hr/tree/13.0/hr_expense_cancel>`_ project on GitHub.
```

Comment:
Please change :

This module is part of the `OCA/hr-expense <https://github.com/OCA/hr-expense/tree/13.0/hr_expense_cancel>`_ project on GitHub.